src/main/java/com/google/summit/ast/expression/SoqlOrSoslExpression.kt

```diff
@@ -20,13 +20,13 @@ import com.google.summit.ast.Node
 import com.google.summit.ast.SourceLocation
 
 /**
- * A SOQL or SOSL expression.
+ * A SOQL expression.
  *
  * @property query the raw string
  * @property bindings the bound expressions
  * @param loc the location in the source file
  */
-class SoqlOrSoslExpression(
```

Comment:
I didn't see any great need to preserve a common base class (other than Expression). If you think this is useful, I can keep it.

Reply:
I don't have much of an opinion. I guess we can wait until/if this is ever needed...